docusaurus/docs/React/core-components/channel-list.mdx

```diff
@@ -90,6 +90,53 @@ re-setting the list state, you can customize behavior and UI.
 | `notification.removed_from_channel`                                                       | Removes channel from list                        | [onRemovedFromChannel](#onremovedfromchannel) |
 | `user.presence.changed`                                                                   | Updates the channel                              | N/A                                           |
 
+## Customizing Event Handlers
+
+The event type the `ChannelList` reacts to and its corresponding behavior can be overridden using the appropriate prop. Let's take an example of ChannelList component for [frozen channels](https://getstream.io/chat/docs/javascript/freezing_channels/).
```

Comment:
this looks like it's missing some words, "Let's look at an example of customizing the `ChannelLIst` component..."